Update german word list

pstienem opened the pull request:
I took the freedom to revise the German word list a little bit to remove English words and German phrases too difficult to draw.
There are no changes to the rest of the code.

Bios-Marcel commented on game/words/en_us:
```diff
@@ -4055,7 +4055,7 @@ skin
 skirt
 skittles
-skribbl.io
+skribbl.rs
```
lol

Bios-Marcel commented:
the word that were article + noun, have you checked whether the standalone nouns already existed? If not, it'd be sad to just lose those words. For example "der verkehr" could just become "verkehr".

pstienem commented:
> 
> 
> the word that were article + noun, have you checked whether the standalone nouns already existed? If not, it'd be sad to just lose those words. For example "der verkehr" could just become "verkehr".

Yes, I did. I added those without article that were not already present and deleted the rest.
